webapp/team.py

```diff
@@ -16,6 +16,19 @@ def get_all_display_names():
     return [person.display_name for person in team.members]
 
 
+def get_all_teams():
+    return [
+        "User research",
+        "Web",
+        "Workplace Engineering",
+        "Apps:  MAAS,  JAAS",
+        "Marketplaces",
+        "Design tribe:  Vanilla,  Brand,  Desktop & installers",
+        "Commercial: Shop, CUBE",
+        "Landscape",
+    ]
```

Comment:
Do we have a better place we could be taking this list from (ideally not a spreadsheet)?